airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -260,6 +267,27 @@ def _wait_for_task_ended(self) -> None:
         waiter.config.max_attempts = sys.maxsize  # timeout is managed by airflow
         waiter.wait(cluster=self.cluster, tasks=[self.arn])
 
+        return
+
+    def _cloudwatch_log_events(self):
+        if not self.awslogs_group:
+            return None
+
+        if not self.awslogs_stream_prefix:
+            return None
```

Comment:
Maybe we want to pull the "requirements" out of the method and do sth like: if requirement than call function _else_ return None. WDYT?

Reply:
Good point, I am not sure if you had something like this in mind:

```py
            if self._aws_logs_enabled():
```